### PR TITLE
Web share updates for M89

### PR DESCRIPTION
**Pull request by jeffposnick**
This updates the Web Share and Web Share Target blog posts to account for changes in Chrome 89, and to make the browser-support story more accurate in general.

R: @jpmedley @petele

### CODE REVIEW

File: src/site/content/en/blog/web-share-target/index.md [hunk 1]
```diff
@@ -206,8 +205,8 @@ window.addEventListener('DOMContentLoaded', () => {
 });
 ```
 
-Be sure to use a service worker to precache the `action` page so that it will
-load quickly and work reliably, even if the user is offline.
+Be sure to use a service worker to [precache](https://web.dev/precache-with-workbox/) the `action`
```
Comment (jpmedley) [inline src/site/content/en/blog/web-share-target/index.md:208]: Someone completely new to this technology might think workbox is the only way to precache. How about linking to an article definition, the appending something like, "an easy way to do this is with workbox."

File: src/site/content/en/blog/web-share-target/index.md [hunk 2]
```diff
@@ -256,6 +256,22 @@ the `text` field, or occasionally in the `title` field.
 
 <div class="w-clearfix"></div>
 
+## Browser support
+
+As of early 2021, the Web Share Target API is supported by:
+
+- Chrome and Edge 76+ on Android.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-share-target/index.md:263]:
```suggestion
- Chrome and Edge 76 or later on Android.
```

File: src/site/content/en/blog/web-share-target/index.md [hunk 3]
```diff
@@ -256,6 +256,22 @@ the `text` field, or occasionally in the `title` field.
...
+- Chrome 89+ on Chrome OS.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-share-target/index.md:264]:
```suggestion
- Chrome 89 or later on Chrome OS.
```

File: src/site/content/en/blog/web-share/index.md [hunk 4]
```diff
@@ -43,14 +43,12 @@ way as platform-specific apps.
...
-  through the `onload` handler is impossible.
-* It can share, URLs, text, or files.
-* As of mid 2020, it is only available on Safari and on Android in Chromium
-  forks. See
-  [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share#Browser_compatibility)
-  for details.
+  through the `onload` handler is not allowed.
+* It can share URLs, text, and in some browsers, files.
+* Some features are only available on specific combinations of browsers and platforms. Please see
+  the [browser support](#browser-support) section for more details.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-share/index.md:51]:
```suggestion
  the [browser support](#browser-support) section for details.
```

File: src/site/content/en/blog/web-share/index.md [hunk 5]
```diff
@@ -138,17 +136,36 @@ with games and educational experiences.
...
+supported.
+
+As of early 2021, using the API to share title, text, and URL is supported by:
+
+- Safari 12+ on macOS and iOS.
+- Chrome 75+ on Android, and 89+ on Chrome OS and Windows.
+
+Using the API to share files is supported by:
+
+- Chrome 75+ on Android, and 89+ on Chrome OS and Windows.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-share/index.md:163]:
```suggestion
- Chrome 75 or later on Android
- 89 or later on Chrome OS and Windows.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/web-share-target/index.md (hunk 1 above)
Comment (jeffposnick) [reply src/site/content/en/blog/web-share-target/index.md]: Changed.

**Comment by jeffposnick** [comment]
Please take another look. Thanks!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**